tasmota/xsns_53_sml.ino

```diff
@@ -1796,7 +1809,7 @@ void SML_Show(boolean json) {
 
 /*
 #ifdef USE_DOMOTICZ
-  if (json && !TasmotaGlobal.tele_period) {
+  if (json && !tele_period) {
```

Comment:
This fails domoticz (I think) as all globals have been put in a struct called TasmotaGlobal